CHANGELOG.md

```diff
@@ -1,5 +1,63 @@
 # Changelog
 
+## [1.0.0] - 2021-07-08
+
+### Added
+- The `vault` UpstreamAuthority plugin now supports Kubernetes service account authentication (#2356)
+- A new `cert-manager` UpstreamAuthority plugin is now available (#2274)
+- SPIRE Server CLI can now be used to ban agents (#2374)
+- SPIRE Server CLI now has `count` subcommands for agents, entries, and bundles (#2128)
+- SPIRE Server can now be configured for SPIFFE federation using the `https_web` and `https_spiffe` profiles (#2340)
```

Comment:
The way that this is worded may be a little confusing, I think. The `https_web` and `https_spiffe` profiles were supported previously also, but the configuration was not explicit about using those names. So maybe I would reword this to reflect the fact that the configuration format for federation relationships was changed to be more consistent with the language expressed in the SPIFFE Federation spec.